docs/manual/docs/cmd/pa/pcf/pcf-init.md

```diff
@@ -0,0 +1,39 @@
+# pa pcf init
+
+Initializes a directory with a new PowerApps component framework project
+
+## Usage
+
+```sh
+pa pcf init [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`--name <name>`|The name for the component.
+`--namespace <namespace>`|The namespace for the component.
+`--template [template]`|Choose a template for the component. Field|Dataset.
```

Comment:
`Fields|Dataset` breaks table formatting. To avoid it, it should be escaped as code (see the `output` option)